src/docs/environment/index.mdx

```diff
@@ -292,3 +298,24 @@ In your sentry virtualenv:
 pip uninstall uwsgi
 pip install --no-cache-dir uwsgi
 ```
+
+---
+
+**Problem:** `DoesNotExist: Subscription matching query does not exist`
+
+**Solution:** In getsentry run the following to mock a subscription:
+
+```bash
+./bin/mock-subscription <org_slug>
+```
+
+---
+
+**Problem:** Something like `Error: No such container: sentry_postgres`
+
+**Solution:** Review the [bootstrap services](http://localhost:8000/environment/#bootstrap-services)
```

Comment:
```suggestion
**Solution:** You need <Link to="/services/devservices/">devservices</Link>.
```

Reply:
After you merge this PR I think it's better to just bake the link to develop docs inside of sentry to make that error more helpful. I can follow up with that.

Reply:
Oh wait, I didn't see the remaining text. Just change the link away from localhost.

Reply:
lol will fix